### PR TITLE
Add comment to top and bottom of generated content

### PR DESCRIPTION
**Pull request by arnested**
One of my coworkers was surprised that his manual additions on a server kept disappearing.

He suggested to add some "generated file" comment.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -62,6 +62,11 @@ func main() {
 					Name:  "user",
 					Usage: "Return given user public ssh keys (this option can be used multiple times for multiple users)",
 				},
+				cli.StringFlag{
+					Name:  "comment",
+					Usage: "Include comment",
+					Value: "Generated by fetch-ssh-keys - DO NOT EDIT",
```
Comment (ernoaapa) [inline main.go]: Should we default to not write the comment? And if you want some comment, you add the —comment flag. So if users upgrade and keep the old flags, the functionality stays same.

**Review by ernoaapa — CHANGES_REQUESTED** [review]
Thank you! Looks great just one comment about default value.

File: main.go (hunk 1 above)
Comment (arnested) [reply main.go]: I've been having the same thought after submitting the PR. I guess that'll make it a good idea :-)

Stay tuned.

**Comment by arnested** [comment]
* Made the default not to include a comment.
* Made the comment option a global option instead of an option for the github commmand
* Made one-line comment instead of multi line commet

**Review by ernoaapa — APPROVED** [review]

**Comment by arnested** [comment]
Thank you, @ernoaapa. Any idea why Travis hasn't build a v1.1.6 release from your tag?

**Comment by ernoaapa** [comment]
Yeah I was wondering same thing. I thought that there’s some delay with the web hooks but probably there’s something else. I’ll check this more soon.

**Comment by ernoaapa** [comment]
@arnested now it's building, had to re-create the tag to trigger the build. Soon available in https://github.com/ernoaapa/fetch-ssh-keys/releases

**Comment by arnested** [comment]
Cool, thank you!

Sometimes it can be sufficient to go into Settings > Webhooks, edit the hook and redeliver the events (at least that has worked for me with some other integrations).

**Pull request merged**